Dntc.Common/OpCodeHandling/Handlers/CallHandlers.cs

```diff
@@ -111,6 +116,9 @@ public OpCodeHandlingResult Handle(HandleContext context)
 
             if (IgnoredMethods.Contains(methodId))
             {
+                context.ExpressionStack.Pop(1);
```

Comment:
@KallDrexx - please can you check this is ok…?

Reply:
ok this was needed because of removing the calls to System_object::ctor put an if around it to specifically look for that.

otherwise it emitted `return __this` from the ctor… maybe there is a better way to solve this?

Reply:
This shouldnt be hard coded but should pop the number of arguments the call takes.

Reply:
So if we have an ignored function that takes two arguments then it would be a pop 2.  And if it's a function with zero arguments it doesn't pop anything

Reply:
fixed